container/images/python/Dockerfile

```diff
@@ -0,0 +1,111 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# build-pack (dependencies)
+RUN set -ex; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends \
```

Comment:
There must be an easier, cleaner way. 
I really want the Dockerfiles to be easy to read and manage.

Reply:
I agree with you on this, lots of ceremony to it to install, I can try using apt-get to install python, pip and other requirements, but not sure if we will get the versions we need

Reply:
With Python, we can use any version of Python 2.7+. For Python 3 we need to make some changes to the runner to make it compatible.